submissions/AnnaStoyano/memory-pair-game/index.js

```diff
@@ -0,0 +1,152 @@
+class Game {
+    constructor(cards) {
+        this.container = document.querySelector('.container');
+        this.cards = cards;
+    }
+
+    render() {
+        this.cards.forEach(card => this.container.insertAdjacentElement('beforeend', card.render()));
+    }
+
+    checkEqualCards(activeCards) {
+        const activeCardsNames = activeCards.map(card => card.cardForDisplay.dataset.name);
+        if (activeCardsNames[0] === activeCardsNames[1]) {
+            activeCards.forEach(card => card.addOpened());
+            return true;
+        }
+        return false;
+    }
+
+    removeActiveCards(cards, isOpened) {
+        let time = isOpened ? 0 : 600;
+        setTimeout(function () {
+            cards.forEach(card => card.removeActive());
+        }, time);
+        return [];
+    }
+
+    processClickOnCards() {
+        const game = this;
+        this.container.addEventListener('click', function (e) {
```

Comment:
you can destructure your event in the function argument 
```suggestion
        this.container.addEventListener('click', function ({target}) {
```